lib/mediawiki/page.rb

```diff
@@ -24,16 +26,33 @@ def replace_output(new_content, run_comment = '')
         client.edit(title: title, text: new_wikitext)
       end
 
+      def param_key_value_array(key, value)
```

Comment:
Is this meant to be part of the public interface? If it's only expected to be called from within `params`, it's probably better to make it private.